main.js

```diff
@@ -65,8 +65,8 @@ const input = cli.input;
 const opts = cli.flags;
 
 const log = report => {
-	// Legacy
-	// TODO: Remove in 1.0.0
+  // Legacy
+  // TODO: Remove in 1.0.0
```

Comment:
I do not understand why XO does not complain about that change :confused: – Please revert it.